fancy-settings/source/settings.js

```diff
@@ -11,7 +11,7 @@ function localMessageHandler(msg, port) {
 
 // Test for the presence of an Edit Server
 function test_server() {
-	var port = chrome.extension.connect();
+	var port = browser.runtime.connect();
```

Comment:
This breaks on Chrome: Uncaught ReferenceError: browser is not defined
    at Object.test_server (settings.js:14)
    at Object.<anonymous> (mootools-core.js:1436)
    at Array.forEach (<anonymous>)
    at Function.forEach (mootools-core.js:211)
    at Array.each (mootools-core.js:324)
    at Object.fireEvent (mootools-core.js:1434)
    at Object.wrapper.extend.$owner (mootools-core.js:1316)
    at Object.<anonymous> (setting.js:161)
    at HTMLInputElement.defn (mootools-core.js:3703)

So maybe this isn't as cross-platform as it should be